rosdep/base.yaml

```diff
@@ -3849,11 +3849,6 @@ libzip-dev:
   fedora: [libzip-devel]
   gentoo: [dev-libs/libzip]
   ubuntu: [libzip-dev]
-libzmq3-dev:
```

Comment:
Removing this key will break anyone using it.

Reply:
Is it possible to get an overview of which packages make use of this key? Otherwise, could we just forward this key to another, so that at least we do not need to maintain both (i.e. keep the package names up to date)?